internal/server/http/handler/v1/tree/handler_test.go

```diff
@@ -0,0 +1,408 @@
+package tree
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"net/http"
+	"net/http/httptest"
+	"strconv"
+	"testing"
+
+	"github.com/gofiber/fiber/v2"
+	"github.com/green-ecolution/green-ecolution-backend/internal/entities"
+	httpEntities "github.com/green-ecolution/green-ecolution-backend/internal/server/http/entities"
+	"github.com/green-ecolution/green-ecolution-backend/internal/service"
+	serviceMock "github.com/green-ecolution/green-ecolution-backend/internal/service/_mock"
+	"github.com/green-ecolution/green-ecolution-backend/internal/storage"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/mock"
+)
+
+func TestGetTreeByID(t *testing.T) {
+	t.Run("should return tree successfully", func(t *testing.T) {
+		app := fiber.New()
+		mockTreeService := serviceMock.NewMockTreeService(t)
+		app.Get("v1/tree/:id", GetTreeByID(mockTreeService))
+
+		treeID := int32(1)
+		mockTree := &entities.Tree{
```

Comment:
Can this also be data from the utils?

Reply:
I am going to approve the MR, but please check before merging, if you can use the test data here.